radix-engine/src/model/resource.rs

```diff
@@ -571,6 +571,31 @@ impl LockableResource {
     }
 }
 
+impl Into<Resource> for &LockableResource {
```

Comment:
is there a way to reuse this (`&LockableResource`) impl for non-ref `LockableResource` (with an extra `is_locked` check), without unnecessary cloning?

Reply:
_(removed this `Into<Resource>` impl)_